inject-java/src/test/groovy/io/micronaut/annotation/mapping/ReplacesRepeatableAnnotationSpec.groovy

```diff
@@ -18,22 +16,21 @@ class ReplacesRepeatableAnnotationSpec extends AbstractTypeElementSpec {
             def definition = buildBeanDefinition('addann.ReplaceAnnotationsTo', '''
 package addann;
 
-import io.micronaut.context.annotation.Requires;
 import io.micronaut.context.annotation.Bean;
 
-@Requires(property = "foo")
-@Requires(property = "bar")
+@io.micronaut.annotation.mapping.MyRequires(property = "foo")
+@io.micronaut.annotation.mapping.MyRequires(property = "bar")
```

Comment:
can you add a new test instead of replacing the existing one and changing it?

Reply:
I just modified it to use a custom annotation instead of the requires that has a repeatable container entry in `AnnotationMetadataSupport`. The test was added by me a few PRs ago it's not something that old.